File: src/addOdd.js

/*
Input: [4 , 3 , 5]
Output: 8
*/

export const addOdd= (arr) => {
   var sum =0;
	for (i=0;i<arr.length;i++){
      x= arr[i];
		if(x%2!=0){
          sum= sum+x;
        }
			
    }
    return sum;
};

File: src/addOdd.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int arr[10]={1,3,2,4,5,3,2,6,8,9};
    int sum = 0;
    for(int i = 0; i < 10; i++){
        if(arr[i]%2){
            sum+=arr[i];
        }
    }
    cout<<sum<<endl;
    return 0;
}
